tools/opt/opt.cpp

```diff
@@ -214,6 +213,9 @@ Options (in lexicographical order):
                growth threshold. The threshold prevents the loop peeling
                from happening if the code size increase created by
                the optimization is above the threshold.
+  --max-id-bound=<n>
+               Sets the maximum value for the id bound for the moudle.  The
+               default is the minimum value for this limit, 4,194,303.
```

Comment:
Reference the spec here?  And make consistent with the hex notation used in the error message.

Reply:
done.  I've had to rebase because of the new spir-v header files.